src/main/java/org/example/Police.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Hello world!
+ */
+public class Police {
+
+    private static final Integer SAFE_PASSWORD = 1000;
+
+    public static boolean captured = true;
```

Comment:
you can use the System.exit(0) method when the combination is found or time runs out